plugins/node/opentelemetry-instrumentation-aws-sdk/test/sqs.test.ts

```diff
@@ -55,6 +55,10 @@ import * as sinon from 'sinon';
 import * as messageAttributes from '../src/services/MessageAttributes';
 import { AttributeNames } from '../src/enums';
 
+// set aws environment variables, so tests in non aws environment are able to run
+process.env.AWS_ACCESS_KEY_ID = 'testing';
+process.env.AWS_SECRET_ACCESS_KEY = 'testing';
+
```

Comment:
Reviewer note: Now that .tav.yml is running this test file alone (`mocha ... test/sqs.test.ts`), instead of with all the other test files (in the same process), this now needs to set `AWS_` auth-related envvars to avoid a client error. When running *all the test files* (`mocha ... 'test/**/*.test.ts'`), then those envvars were being set in "test/aws-sdk-v3.test.ts" and solving the auth-related issue for all the test files).